src/tree/CosmosDBAccountProvider.ts

```diff
@@ -30,10 +30,19 @@ export class CosmosDBAccountProvider implements IChildProvider {
     public async loadMoreChildren(node: IAzureNode): Promise<IAzureTreeItem[]> {
         const client = new CosmosDBManagementClient(node.credentials, node.subscriptionId);
         const accounts: DatabaseAccountsListResult = await client.databaseAccounts.list();
+        let accountTreeItems = [];
+        await Promise.all(
+            accounts.map(async (databaseAccount: DatabaseAccount) => {
+                try {
+                    let account = await this.initChild(client, databaseAccount);
+                    accountTreeItems.push(account);
+                } catch (e) {
+                    accountTreeItems.push(<IAzureTreeItem>{ id: e.toString(), label: e.toString() });
```

Comment:
label: That's what parseError is for.  e could be anything - a string, a number, even null

id: If you get the same error for multiple nodes, you'll get duplicate ID's, which I think causes vscode to throw.  Does Eric's PR have this issue?

Reply:
Just leave the id out - it's optional.

But even then, it would be weird to have multiple labels with the same message. I would prefer if the label was still the account name and the description was 'invalid'. If you want to display the error message, I would make it a child node.

(If we don't even know the account name, I just wouldn't display the node in the tree. I don't think there's much we can do there)